proof/crefine/RISCV64/Ctac_lemmas_C.thy

```diff
@@ -46,6 +46,16 @@ lemma c_guard_abs_sc:
 
 lemmas ccorres_move_c_guard_sc[corres_pre] = ccorres_move_c_guards[OF c_guard_abs_sc]
 
+lemma c_guard_abs_reply:
+  fixes p :: "reply_C ptr"
+  shows "\<forall>s s'. (s, s') \<in> rf_sr \<and> reply_at' (ptr_val p) s \<and> True \<longrightarrow> s' \<Turnstile>\<^sub>c p"
+  apply clarsimp
+  apply (drule (1) reply_at_h_t_valid)
+  apply simp
+  done
```

Comment:
probably can one-line it with `clarsimp dest: reply_at_h_t_valid` (or if that doesn't work, `dest!`, or if that doesn't work use `dest!` and fastforce)

Reply:
Done.

Reply:
That was fast. Thanks